library/src/main/java/com/novoda/downloadmanager/DownloadMigrator.java

```diff
@@ -0,0 +1,60 @@
+package com.novoda.downloadmanager;
+
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.Handler;
+import android.os.IBinder;
+
+public class DownloadMigrator {
```

Comment:
The clients only interact with this class and the associated `builder` now.

Reply:
Shall we expose then a public interface? ;)